docs/source/conf.py

```diff
@@ -2454,6 +2454,8 @@
     "SharedQuantizationSpec",
     # torch.ao.quantization.quantizer.x86_inductor_quantizer
     "X86InductorQuantizer",
+    # torch.ao.quantization.quantizer.xpu_inductor_quantizer
+    "XPUInductorQuantizer",
```

Comment:
I'm not sure if we want to add more things to this list actually, removing and fix the lint might be better, if it's possible

Reply:
This line is wrongly added by me and I have removed it in PR, great thanks for your reminding.

Reply:
these are saying you should addd docs in .rst file, seems reasonable request I feel, do you want to add docs for this quantizer in https://pytorch.org/docs/stable/quantization-support.html

Reply:
maybe you can add something similar to https://github.com/pytorch/pytorch/blob/main/docs/source/quantization-support.rst#torchaoquantizationquantizer in the same file if possible, otherwise we can just land, this is not too important right now I think